newlib/libc/sys/vita/crt0.c

```diff
@@ -3,7 +3,7 @@
 #include <stdlib.h>
 
 /* The maximum number of arguments that can be passed to main(). */
-#define ARGC_MAX 19
+#define ARGC_MAX 31
```

Comment:
@isage did you tested this? maybe it related `sceAppMgrLoadExec` function.

@frangarcj  can you review this? it related #13

Reply:
Yes. 31 comes from reversing sce crt0 (19 is a psp limit)
(sceAppMgrLoadExec actually additionally limits params to 1024 chars)